test/stubs/discord-stub.js

```diff
@@ -12,6 +12,9 @@ export function getChannel(key, value) {
 class DiscordStub extends events.EventEmitter {
   constructor() {
     super();
+    this.users = {
+      get: sinon.stub()
```

Comment:
Maybe this should be a sandbox stub, like the ones defined here? https://github.com/rce/discord-irc/blob/irc-to-discord-mentions/test/bot.test.js#L29 That way it gets reset after each test, here: https://github.com/rce/discord-irc/blob/irc-to-discord-mentions/test/bot.test.js#L38